subprojects/packagefiles/whisper_cpp/meson.build

```diff
@@ -0,0 +1,66 @@
+project('whisper.cpp', ['cpp', 'c'], version: '1.5.2')
+
+cc = meson.get_compiler('c')
+
+math = cc.find_library('m', required: false)
+threads = dependency('threads')
+
+if build_machine.system() == 'linux'
+    add_project_arguments('-D_GNU_SOURCE', language: 'c')
+
+    if not get_option('no_avx')
```

Comment:
Don't do double negation, e.g. `if get_option('avx').enabled() ..,`

Reply:
And I would prefer x86 extensions opt-in, rather than opt-out. Chances of it being compiled elsewhere are high.
At least check for `host_machine.cpu_family() == 'x86_64'`

Reply:
I'll switch these around, this was just a one for one port of the original cmake build files, I didn't want to mess with things too much